pom.xml

```diff
@@ -266,56 +334,109 @@
+                                    <arg value="git describe --abbrev=0 | tr -d v"/>
+                                </exec>
+                                <echo>Full Version: ${fullVersion}</echo>
                                 <echo>Version: ${releaseVersion}</echo>
-							</target>
-							<exportAntProperties>true</exportAntProperties>
-						</configuration>
-					</execution>
-					<execution>
-						<phase>package</phase>
-						<goals>
-							<goal>run</goal>
-						</goals>
-						<configuration>
-							<tasks>
-								<!-- Create plugin .bar -->
-								<jar destfile="${project.build.directory}/NscSdnControllerPlugin.bar">
-									<fileset dir="${project.build.directory}/plugin" />
-									<manifest>
-										<attribute name="Deployment-Name" value="NSC" />
-										<attribute name="Deployment-SymbolicName" value="NscSdnControllerPlugin" />
-                                        <attribute name="Deployment-Version" value="${fullVersion}" />
-										<attribute name="Require-Bundle" value="sdn-controller-nsc-plugin" />
-									</manifest>
-								</jar>
-							</tasks>
-						</configuration>
+
+                            </target>
+                            <exportAntProperties>true</exportAntProperties>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                        <configuration>
+                            <tasks>
+                                <!-- Create plugin .bar -->
+                                <jar destfile="${project.build.directory}/NscSdnControllerPlugin.bar">
+                                    <fileset dir="${project.build.directory}/plugin"/>
+                                    <manifest>
+                                        <attribute name="Deployment-Name" value="NSC"/>
+                                        <attribute name="Deployment-Type" value="SDN"/>
+                                        <attribute name="Deployment-SymbolicName" value="NscSdnControllerPlugin"/>
+                                        <attribute name="Deployment-Version" value="${fullVersion}"/>
+                                        <attribute name="Require-Bundle" value="sdn-controller-nsc-plugin"/>
+                                    </manifest>
+                                </jar>
+                            </tasks>
+                        </configuration>
+                    </execution>
+                       
+					 <execution>
+					  <id>h2-fragment</id>
+					  <phase>prepare-package</phase>
+					  <goals>
+					    <goal>run</goal>
+					  </goals>
+					  <configuration>
+					    <tasks>
+					      <jar destfile="C:\Users\dgerenrx\prog\osc\sdn-controller-nsc-plugin\target/plugin/h2ServiceExport.jar">
```

Comment:
not sure what all the fragment references are doing here.

Reply:
These are from security manager .

Reply:
ahh...k. Please remove references to your user directory.C:\Users\dgerenrx\prog\osc\sdn-controller-nsc-plugin\target